teamserver/teamserver/events/worker.py

```diff
@@ -7,6 +7,7 @@
 from mongoengine import connect
 
 from teamserver.integrations import Integration, SlackIntegration
```

Comment:
Can be placed on same line as other imoirts

Reply:
Breaks 80char limit. Will do though